### PR TITLE
Release 1.23 post

### PR DESCRIPTION
**Pull request by czoido**

### CODE REVIEW

File: _posts/2020-03-13-New-conan-release-1-23.markdown [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+layout: post 
+comments: false 
+title: "Conan 1.23: Parallel installation of binaries, CONAN_V2_MODE environment variable to enable
+Conan v2 behavior and more"
+---
+
+March brings us a new Conan release but sadly it also brought us the notice of having to postpone
+ConanDays due to growing concerns about the Coronavirus. This was a hard decision to make for the
+safety and well being of all of you, but we hope it will make possible for an even greater meeting of
+the Conan community soon. But let's focus on the good news and see what comes with Conan 1.23.
```
Comment (danimtb) [inline _posts/2020-03-13-New-conan-release-1-23.markdown]: 💯

File: _posts/2020-03-13-New-conan-release-1-23.markdown [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+As you can see it just took around 30 seconds to download all the binaries what means an
+**improvement of around a 600% in time**.
+
+If you try this feature you may experience some message overlaps in the command line output. We have
+prioritized the feature over a clean output but will solve these output problems shortly.
+
+Use all of these features in combination or separate and please tell us if they are making your CI's
+faster.
+
+## CONAN_V2_MODE to start testing Conan v2 deprecated features
```
Comment (jgsogo) [inline _posts/2020-03-13-New-conan-release-1-23.markdown]: Maybe it is just me, but I'd change the _"start testing Conan v2 deprecated features"_ in my mind it looks like Conan v2 has already deprecated features 🙈

File: _posts/2020-03-13-New-conan-release-1-23.markdown [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+the deprecation of features for Conan 2.0. We have introduced the `CONAN_V2_MODE` [environment
+variable](https://docs.conan.io/en/latest/reference/conan_v2_mode.html#conan-v2-mode) that activates
+some behaviors and defaults that are intended to be in the next major release.
+
+Some of the most important default behaviours for Conan 2.0 will be:
+
+* Revisions are enabled by default (adds `revisions_enabled=1` to *conan.conf*).
+* No hooks activated by default.
+* SCM data will be stored into *conandata.yml*.
+* GCC >= 5 autodetected profile will use `libstdc++11`.
```
Comment (jgsogo) [inline _posts/2020-03-13-New-conan-release-1-23.markdown]: Add something like, _we will keep adding more things to this mode_

File: _posts/2020-03-13-New-conan-release-1-23.markdown [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+faster.
+
+## CONAN_V2_MODE to start testing features that will deprecate when Conan v2 is released
+
+Although we still have plenty of time and work ahead before Conan 2.0 we would like to start testing
+the deprecation of features for Conan 2.0. We have introduced the `CONAN_V2_MODE` [environment
+variable](https://docs.conan.io/en/latest/reference/conan_v2_mode.html#conan-v2-mode) that activates
+some behaviors and defaults that are intended to be in the next major release.
+
+Some of the most important default behaviours for Conan 2.0 will be:
```
Comment (uilianries) [inline _posts/2020-03-13-New-conan-release-1-23.markdown]:
```suggestion
Some of the most important default behaviors for Conan 2.0 will be:
```

File: _posts/2020-03-13-New-conan-release-1-23.markdown [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+* Revisions are enabled by default (adds `revisions_enabled=1` to *conan.conf*).
+* No hooks activated by default.
+* SCM data will be stored into *conandata.yml*.
+* GCC >= 5 autodetected profile will use `libstdc++11`.
+
+We will be adding behaviours to this mode as we progress in the roadmap to Conan v2 so stay updated
```
Comment (uilianries) [inline _posts/2020-03-13-New-conan-release-1-23.markdown]:
```suggestion
We will be adding behaviors to this mode as we progress in the roadmap to Conan v2 so stay updated
```

File: _posts/2020-03-13-New-conan-release-1-23.markdown [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+Our objective is to minimize the impact on existing recipes when Conan 2.0 is released and start
+gathering feedback about the new configuration and behavior. **Be advised that this mode is only for
+experimenting, please do not activate this mode in a production environment!** 
+
+## Other cool things
+
+ * `clean-modified` subcommand for `graph` command. When a package of a dependency graph is going
+   to be re-built, using a given lockfile, it is desired to finish the build knowing which packages
+   of the graph have been rebuilt as a result of the last command. This command will clean
+   all the previously existing "modified" flags before such build, so after the build the "modified"
```
Comment (uilianries) [inline _posts/2020-03-13-New-conan-release-1-23.markdown]:
```suggestion
   all the previously existing "modified" flags before such build, so after the build, the "modified"
```

**Comment by uilianries** [comment]
Screenshot_2020-03-12 Conan 1 23 Parallel installation of binaries, CONAN_V2_MODE environment variable to enable Conan v2 b

**Pull request merged**